packages/yoastseo/src/scoring/assessments/seo/SingleH1Assessment.js

```diff
@@ -10,10 +10,10 @@ import AssessmentResult from "../../../values/AssessmentResult.js";
 import Mark from "../../../values/Mark.js";
 
 /**
- * Assessment to check whether the body of the text contains H1s in the body (except at the very beginning,
- * where they are acceptable).
+ * Assessment to check whether the body of the text contains more than 1 H1s in the body.
+ * This assessment doesn't penalize H1 that is not in the very beginning of the body.
  */
-class singleH1Assessment extends Assessment {
+class SingleH1Assessment extends Assessment {
 	/**
```

Comment:
Changed the first letter to upper case to follow Javascript naming convention for Class.